### PR TITLE
Change over images to new higher resolution format and display in all instances

### PR DESCRIPTION
**Pull request by Rabid-Dan**
User story from Ackama jira ticket NZSL-26

```
As a user

I want to see images that are un cropped and have original dimensions of 400 x 400

So that the image quality is higher than its current state
```

Historically images have needed to be cropped due to some offset issues. More recently a new set of clearer images, that have a max width or height of 400px and dont require cropping, have been created and uploaded to freelex for use. 

Ideally the NZSL team would like to use these as the default images for everything if possible so this branch changes all instances of images pulled from freelex to the new dimensions.

Due to the increase in size and clarity we'll need to do some benchmarking to make sure that page load times dont increase dramatically. The new images are larger but are still not huge, so load time stats will be taken against staging before this branch is merged and then again after this has been merged for comparison.

This piece of work also means that a bunch of old helper methods can be removed as they were previously relied on for calculating image dynamic image dimensions in the vocab sheet and have now been removed.

### CODE REVIEW

**Comment by Rabid-Dan** [comment]
I'll take a second pass on this tomorrow morning and maybe add some more tests, but reviews on the initial implementation are welcome.

File: app/services/image_processor.rb [hunk 1]
```diff
@@ -14,24 +14,16 @@ def initialize(filename:, height:, width:)
...
     return @local_filename if local_file_exists && local_file_age_in_days < 1
-    resize
+    write_file_locally
     @local_filename
   end
 
   private
 
-    def resize
+    def write_file_locally
```
Comment (Br3nda) [inline app/services/image_processor.rb:25]: noice rename

Comment (louise-r-blue) [reply app/services/image_processor.rb:25]: I agree

**Review by louise-r-blue — APPROVED** [review]
an amazing implementation and tidy up.

File: app/assets/stylesheets/_vocab_sheet.scss [hunk 2]
```diff
@@ -196,19 +196,31 @@
 .vocab_sheet_drawing__size {
   &--2 {
     min-height: 290px;
```
Comment (fractalcactus) [inline app/assets/stylesheets/_vocab_sheet.scss]: my scss is rudimentary - so 🐘 , but this repetition of 
```max height: number &img max-height: number - 10 ``` could be dry'd up with a mixin?

File: app/controllers/sign_image_controller.rb [hunk 3]
```diff
@@ -25,13 +25,13 @@ def filename_param
     def width_param
       sign_image_params[:width].to_i
     rescue StandardError
-      100
+      400
```
Comment (fractalcactus) [inline app/controllers/sign_image_controller.rb:28]: nice!

File: app/assets/stylesheets/_vocab_sheet.scss (hunk 2 above)
Comment (Rabid-Dan) [reply app/assets/stylesheets/_vocab_sheet.scss]: oooh thats a great idea thanks!

**Review by fractalcactus — APPROVED** [review]

**Review by KatieWright26 — APPROVED** [review]

File: app/helpers/vocab_sheet_helper.rb [hunk 4]
```diff
@@ -5,30 +5,6 @@ def vocab_sheet?
...
-  end
-
-  def vocab_sheet_label_fontsize
-    @vocab_sheet_label_fontsize ||= (vocab_sheet_label_height.to_f / 2).floor
-  end
-
-  def vocab_sheet_item_width
-    @vocab_sheet_item_width ||= vocab_sheet_image_width + 5
-  end
-
```
Comment (ghostPepperWonton) [inline app/helpers/vocab_sheet_helper.rb:31]: Aaaawww yeaaaaah, dropping these is so exciting

**Review by ghostPepperWonton — APPROVED** [review]

**Review by Br3nda — APPROVED** [review]

**Pull request merged**